### DISCUSSION TITLE
New Release?

### DISCUSSION THREAD

**Discussion by ldinino** [trunkmaster/nextspace#410] [Q&A] [answered]
Trunkmaster,

I’m really loving the work that you’ve put into this project. I can really tell it’s a passion project of yours. I love the attention to detail and just how accurate the UI and UX is to NextStep and OpenStep. 

Do you think you’ll be making another release at some point? Possibly with support for newer and supported releases of RHEL like Rocky Linux? 

Possibly even other more user-friendly and common distros like Ubuntu? (Assuming not likely)

**Comment by trunkmaster** [comment, marked as answer]
@ldinino,
> I’m really loving the work that you’ve put into this project. I can really tell it’s a passion project of yours. I love the attention to detail and just how accurate the UI and UX is to NextStep and OpenStep.

Thank you!

> Do you think you’ll be making another release at some point? 

Sure. I need to finish my effort to implement and test new notifications mechanism and make X11 applications closer to being first class citizens in NeXT world - applications menus. There are some bugs need to fixed also.

> Possibly with support for newer and supported releases of RHEL like Rocky Linux?

Technically it is quite possible. But I've decided to stop at CentOS 7 (still supported version) and Fedora (my current development environment).

> Possibly even other more user-friendly and common distros like Ubuntu? (Assuming not likely)

Sorry, I have lack of knowledge of Debian-based packaging machinery. I'd better write some more code. ;)
Although, there's some code (unfinished I guess) for building .debs in project's source tree (Packaging/Debian).

**Comment by zeroheure** [comment]
@ldinino  armm77 packages are build for Centos 8 stream
https://github.com/armm77/nextspace-packages
